app/models/web_hook_failures.rb

```diff
@@ -50,7 +50,8 @@ def first
   # reads all elements in the list
   #
   def all
-    _all.map { |e| WebHook::Failure.parse(e) }
+    collection = _all.map { |e| WebHook::Failure.parse(e) }
+    System::Database.postgres? ? collection.sort_by(&:id) : collection
```

Comment:
Same as https://github.com/3scale/porta/pull/376/files#r249718201
In this case that would be:
```ruby
def all
  _all.map { |e| WebHook::Failure.parse(e) }.sort_by(&:id)
end
```

Reply:
Do we need to sort that? It is retrieved from redis right?